components/google-cloud/google_cloud_pipeline_components/aiplatform/__init__.py

```diff
@@ -17,18 +17,37 @@
 from google_cloud_pipeline_components.aiplatform import utils
 
 __all__ = [
-    'ImageDatasetCreateOp', 'TabularDatasetCreateOp', 'TextDatasetCreateOp',
-    'VideoDatasetCreateOp', 'ImageDatasetExportDataOp',
-    'TabularDatasetExportDataOp', 'TextDatasetExportDataOp',
-    'VideoDatasetExportDataOp', 'ImageDatasetImportDataOp',
-    'TextDatasetImportDataOp', 'VideoDatasetImportDataOp',
-    'CustomContainerTrainingJobRunOp', 'CustomPythonPackageTrainingJobRunOp',
-    'AutoMLImageTrainingJobRunOp', 'AutoMLTextTrainingJobRunOp',
-    'AutoMLTabularTrainingJobRunOp', 'AutoMLVideoTrainingJobRunOp',
-    'ModelDeployOp', 'ModelBatchPredictOp', 'ModelUploadOp', 'EndpointCreateOp'
+    'ImageDatasetCreateOp',
+    'TabularDatasetCreateOp',
+    'TextDatasetCreateOp',
+    'VideoDatasetCreateOp',
+    'ImageDatasetExportDataOp',
+    'TabularDatasetExportDataOp',
+    'TextDatasetExportDataOp',
+    'VideoDatasetExportDataOp',
+    'ImageDatasetImportDataOp',
+    'TextDatasetImportDataOp',
+    'VideoDatasetImportDataOp',
+    'CustomContainerTrainingJobRunOp',
+    'CustomPythonPackageTrainingJobRunOp',
+    'AutoMLImageTrainingJobRunOp',
+    'AutoMLTextTrainingJobRunOp',
+    'AutoMLTabularTrainingJobRunOp',
+    'AutoMLVideoTrainingJobRunOp',
+    'ModelDeployOp',
+    'ModelBatchPredictOp',
+    'ModelUploadOp',
+    'EndpointCreateOp',
+    'TimeSeriesDatasetCreateOp',
+    'TimeSeriesDatasetExportDataOp',
+    'AutoMLForecastingTrainingJobRunOp',
 ]
 
 ImageDatasetCreateOp = utils.convert_method_to_component(
+    aiplatform_sdk.TimeSeriesDataset, aiplatform_sdk.TimeSeriesDataset.create
+)
+
+TimeSeriesDatasetCreateOp = utils.convert_method_to_component(
     aiplatform_sdk.ImageDataset, aiplatform_sdk.ImageDataset.create
```

Comment:
Are the names switched between `ImageDatasetCreateOp ` and `TimeSeriesDatasetCreateOp` seems like `TimeSeriesDatasetCreateOp` contains the relevant things for `ImageDataset`?

```suggestion
TimeSeriesDatasetCreateOp = utils.convert_method_to_component(
    aiplatform_sdk.TimeSeriesDataset, aiplatform_sdk.TimeSeriesDataset.create
)

ImageDatasetCreateOp = utils.convert_method_to_component(
    aiplatform_sdk.ImageDataset, aiplatform_sdk.ImageDataset.create
```

Reply:
Good catch! updating shortly.

Reply:
Done.